buildenv/jenkins/docker-slaves/jenkinsfile-build-container

```diff
@@ -0,0 +1,85 @@
+/*******************************************************************************
+ * Copyright (c) 2018, 2018 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ 
+*******************************************************************************/
+
+def ARCH = (params.ARCH) ? params.ARCH : "unknown"
+def OS = (params.OS) ? params.OS : "unknown"
+if(params.ghprbPullId) {
+    PARSE = "${ghprbCommentBody}".toLowerCase().split()
+    for(i = 0; i < PARSE.length - 4; i++) {
+        if(PARSE[i] == "jenkins" && PARSE[i+1] == "build" && PARSE[i+2] == "docker") {
+            ARCH = PARSE[i+3]
+            OS = PARSE[i+4]
+            break
+        }
+    }
+}
+if((ARCH == "x86" || ARCH == "s390x" || ARCH == "ppc64le") && OS != "unknown") {
```

Comment:
Suggest we have an ARCH array and use `array.containsValue("${ARCH}")`. That way we don't have to change the `if` when more archs are added.